actions/actions.py

```diff
@@ -601,6 +601,11 @@ async def submit(
                 "User-Agent": "rasa/covid19-healthcheckbot",
             }
 
+            # For dev purposes only, set the env for MSISDN since rasa shell uses uuid
+            # which fails validation against the eventstore
+            if config.DEV_MSISDN:
+                post_data.update({"msisdn": config.DEV_MSISDN})
+
```

Comment:
Wasn't sure of a nice way to do this, but this is handy to test submission to the api.

Reply:
You can do a `rasa shell --conversation-id 27820001001`, so I'll remove this

Reply:
🎉